scripts/cassandra/cassandra.cql

```diff
@@ -21,9 +21,12 @@ create table version (
 create table version_successor (
     id bigint PRIMARY KEY,
     from_version_id bigint,
-    to_version_id bigint
+    to_version_id bigint,
+    item_id_set set<bigint>
```

Comment:
Why is this a `set`? Each version successor should only be a part of one `Item`'s DAG. The `VersionHistoryDag` table was meant to be for quick access, but I'm not convinced that that's the case now that I think about it. 

Regardless, there should just be one `Item` id per `VersionSuccessor.`